deploy/resources/monitoring/operator_service_monitor.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    name: knative-serving-operator
+  annotations:
+    networkoperator.openshift.io/ignore-errors: ""
+  name: knative-serving-operator
+spec:
+  endpoints:
+  - interval: 30s
+    port: metrics
+  namespaceSelector: {}
+  selector:
+    matchLabels:
+      name: knative-serving-operator
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
This Role & RoleBinding are duplicated with `service_monitor.yaml`'s. Should we create the separate file and share it between `installServiceMonitor` and `exposeMetricToServiceMonitor`?